lib/scala/project-manager/src/main/scala/org/enso/projectmanager/infrastructure/desktop/DesktopTrash.scala

```diff
@@ -0,0 +1,20 @@
+package org.enso.projectmanager.infrastructure.desktop
+
+import org.enso.desktopenvironment.{Platform, Trash}
+import org.enso.projectmanager.control.effect.Sync
+
+import java.io.File
+
+class DesktopTrash[F[+_, +_]: Sync](trash: Trash) extends TrashCan[F] {
+
+  /** @inheritdoc */
+  override def moveToTrash(path: File): F[Nothing, Boolean] =
```

Comment:
Why do we define yet another interface that has the same API as already existing `Trash`?

Reply:
Please look at the other functionality of PM - this follows the same pattern. It does not mean that I approve it entirely, as it is over-engineered to my taste, but for now we have to live with it. This PR should not refactor the whole interface.

Reply:
OK.